rust/cubestore/cubestore/src/store/mod.rs

```diff
@@ -706,6 +712,118 @@ mod tests {
         let _ = fs::remove_dir_all(chunk_store_path.clone());
         let _ = fs::remove_dir_all(chunk_remote_store_path.clone());
     }
+    #[tokio::test]
+    async fn create_aggr_chunk_test() {
+        let config = Config::test("create_aggr_chunk_test");
+        let path = "/tmp/test_create_aggr_chunk";
+        let chunk_store_path = path.to_string() + &"_store_chunk".to_string();
+        let chunk_remote_store_path = path.to_string() + &"_remote_store_chunk".to_string();
+
+        let _ = DB::destroy(&Options::default(), path);
+        let _ = fs::remove_dir_all(chunk_store_path.clone());
+        let _ = fs::remove_dir_all(chunk_remote_store_path.clone());
+        {
+            let remote_fs = LocalDirRemoteFs::new(
+                Some(PathBuf::from(chunk_remote_store_path.clone())),
+                PathBuf::from(chunk_store_path.clone()),
+            );
+            let meta_store = RocksMetaStore::new(path, remote_fs.clone(), config.config_obj());
+            let chunk_store = ChunkStore::new(
+                meta_store.clone(),
+                remote_fs.clone(),
+                Arc::new(MockCluster::new()),
+                config.config_obj(),
+                10,
+            );
+
+            let col = vec![
+                Column::new("foo".to_string(), ColumnType::String, 0),
+                Column::new("boo".to_string(), ColumnType::Int, 1),
+                Column::new("sum_int".to_string(), ColumnType::Int, 2),
+            ];
+
+            let foos = Arc::new(StringArray::from(vec![
+                "a".to_string(),
+                "b".to_string(),
+                "a".to_string(),
+                "b".to_string(),
+                "a".to_string(),
+            ]));
+            let boos = Arc::new(Int64Array::from(vec![10, 20, 10, 20, 20]));
+
+            let sums = Arc::new(Int64Array::from(vec![1, 2, 10, 20, 5]));
+
+            meta_store
+                .create_schema("foo".to_string(), false)
+                .await
+                .unwrap();
+
+            let ind = IndexDef {
+                name: "aggr".to_string(),
+                columns: vec!["foo".to_string(), "boo".to_string()],
+                multi_index: None,
+                index_type: IndexType::Aggregate,
+            };
+            let table = meta_store
+                .create_table(
+                    "foo".to_string(),
+                    "bar".to_string(),
+                    col.clone(),
+                    None,
+                    None,
+                    vec![ind],
+                    true,
+                    None,
+                    Some(vec![("sum".to_string(), "sum_int".to_string())]),
+                    None,
+                )
+                .await
+                .unwrap();
+
+            let data: Vec<ArrayRef> = vec![foos, boos, sums];
+
+            let indicies = meta_store.get_table_indexes(table.get_id()).await.unwrap();
+
+            let aggr_index = indicies
```

Comment:
Nit: This feels overkill. Perhaps `let aggr_index = indices[0]` or something similar?

Reply:
The fact is that with create_table, the `default` index is automatically created and perhaps some other automatic indexes will be created in the future. The position of the custom index in this list may change with minor edits in the `create_table` implementation and I would not like this to break the tests.